assets/app/views/nav.js

```diff
@@ -20,6 +23,11 @@ var NavbarView = Backbone.View.extend({
     this.$el.html(this.template(user));
 
     return this;
+  },
+  logout: function (e) {
+    e.preventDefault();
+    window.localStorage.removeItem('token');
```

Comment:
I'd use a simple wrapper class for localStorage. It might help a little with testing (although stubbing and mocking is extremely easy in JS). More so, it might help if you want an abstraction on your data. So your data could technically come from local storage or your server, but the view doesn't need to know that. I like the way flux works in this respect, where you have a store for all your data.